index.html

```diff
@@ -814,10 +814,7 @@ <h3 class="title-wrapper center card-header"><a href="https://github.com/ranelpa
                         this.hitsWrong += 1;
                     }
 
-                    if (
-                        typedPhrase.length === this.expectedPhrase.length
-                        && typedPhrase === this.expectedPhrase
-                    ) {
+                    if (typedPhrase.replace(/ +$/, "") === this.expectedPhrase) {
```

Comment:
Please use `.trimEnd()` which is simpler/more readable:
`if (typedPhrase.trimEnd() === this.expectedPhrase) {`